engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py

```diff
@@ -105,7 +105,7 @@ def _warmup(self, **kwargs):
             Batch(id=0, requests=requests,
                   size=len(requests)), self.model.tokenizer,
             kwargs.get("torch_dtype", torch.float16), self.device)
-        self.model.warmup(batch, max_prefill_tokens)
```

Comment:
could we have a logging.warning() message saying `max_prefill_tokens` is deprecated?

Reply:
This option is not deprecated